source/01-global/01-typography/fonts.scss

```diff
@@ -2,6 +2,16 @@
 
 @use '00-config' as *;
 
+@font-face {
+  font-family: 'Source Sans Pro';
+  src: url('../../fonts/source-sans-pro/sourcesanspro-regular-webfont.woff2')
+      format('woff2'),
+    url('../../fonts/source-sans-pro/sourcesanspro-regular-webfont.woff')
+      format('woff'),
+    url('../../fonts/source-sans-pro/sourcesanspro-regular-webfont.ttf')
+      format('ttf');
+}
+
```

Comment:
@kmonahan Should this be commented out as an example since we don’t have these fonts in the theme? Or should we add these font files to Gesso and remove the call to Google Fonts in `.storybook/preview-head.html`?

Reply:
Whoops, that's testing code that's not supposed to be there at all. Thanks for catching! I'll pull that back out.

Reply:
Removed